templates/web/buckinghamshire/report/new/roads_message.html

```diff
@@ -1,8 +1,22 @@
 <div id="js-roads-responsibility" class="box-warning hidden">
-    <strong>Invalid location</strong>
-    <div id="js-not-council-road" class="hidden js-responsibility-message">
-        <p>The selected road is not maintained by Buckinghamshire County Council.</p>
-        <p>Please select another road.</p>
+    <strong>Not maintained by Buckinghamshire County Council</strong>
+    <div id="js-not-council-road-private" class="hidden js-responsibility-message">
```

Comment:
Alternatively to the other comment, I guess this should be `id="js-not-council-road"`?

Reply:
Aha, yes, whoops, ta.